_posts/2020-04-13-squeezing-the-most-out-of-your-pebble.md

```diff
@@ -0,0 +1,132 @@
+[RSS Reader](https://apps.rebble.io/en_US/application/5d8d503bc393f5215a13e5e6) | Read the title and contents of your favourite RSS feeds
+[Hybrid Watchface](https://apps.rebble.io/en_US/application/5e463ef63dd31016ad6b17af) | A minimalistic watchace for Pebble Time and later
+[Sheikah Slate Watchface](https://apps.rebble.io/en_US/application/5e28ea933dd3109f151c7e0a) | The power of the sheikah slate on your wrist!
+
+<br>
+# Timeline
+June last year Rebble implemented timeline support. Which means you can once again push news, data, stocks, events and more to your timeline. It's a convenient way to get information you can about available quickly but unobtrusively. As some you already know, [I](https://willmurphy.co.uk/) wrote [a service](https://willow.systems/integrate-Pebble-with-ifttt-once-again/) that lets you hook anything from [If-This-Then-That](https://ifttt.com) to Pebble timeline, making it easy to push data.    
+
+As an example, I push the following data to my timeline:   
+
+Data  |  Method
+:--|:--
+BBC Breaking News | IFTTT: New Tweet to Timeline
+New post to /r/Pebble | IFTTT: Reddit to Timeline
+New RSS feed | IFTTT: RSS Feed applet to Timeline
+NASA Breaking News alerts | IFTTT: NASA to Timeline
+Monzo Debit card payments | [Timeline Bounce Server](https://github.com/Willow-Systems/timeline-bounce-server)
+New Hot item on hacker news | IFTTT: Hacker News to Timeline
+
+This keeps my timeline full of interesting stuff to read when I choosed to. Only BBC breaking news creates a notification, the rest is passive and I either read it when it pops up in [timeline peek](https://i.redd.it/7l87rqv4y9n41.png) or when I scroll back through timeline past (By mapping the 'up' quicklaunch to timeline past)   
+
+To get going with timeline you'll need [this app to generate a 'token'](https://apps.rebble.io/en_US/application/5d9ac26dc393f54d6b5f5445) and to follow [this guide](https://willow.systems/integrate-Pebble-with-ifttt-once-again/) or - if you prefer pictures - [this guide](https://imgur.com/a/IWG28JA). It helps if you're familiar with [If-This-Then-That](https://ifttt.com) too.
+
+# Developing New Apps
+Do you know how to program in C or Javascript? Maybe you want to learn? Developing on the Pebble is super fun and between the #app-dev channel on [the Discord](https://rebble.io/Discord) and [the documentation](https://developer.rebble.io/developer.Pebble.com/index.html) there's plenty of support still.   
+
+To get started you'll need the SDK. For which there are now a few options:   
+
+   - [Install to your computer natively](https://old.Reddit.com/r/Pebble/comments/9i9aqy/developing_for_Pebble_without_cloudPebble_windows/)   
+   - [Install to your computer with a script](https://gist.github.com/zeevro/ce20d0d74a869d73119e9d1522b7caa7)   
+   - [Use a docker image](https://hub.docker.com/r/dmorgan81/rebble/)   
+   - [Use a virtual machine](https://willow.systems/Pebble/vm/)   
```

Comment:
this link isn't working for me? :link: :mag:

Reply:
There should be no capital in the discord link.

Reply:
Damn pebble -> Pebble find and replace!